static/js/narrow.js

```diff
@@ -381,6 +390,7 @@ exports.narrow_to_next_pm_string = function () {
 
     var opts = {
         select_first_unread: true,
+        hotkey: hotkey,
```

Comment:
I might change the name of this parameter to be something like `force_close`, so it's more explicit that we know in this code box that we want to close the compose box.  (I think it will be true that we want to force_close here even if we eventually used this function for another type of UI, and even if we guess wrong here, we could always fix the code accordingly.)